### ISSUE TITLE
Can't retrieve the list of servers from speedtest.net

### ISSUE THREAD

**Issue by tribet84** [wasbazi/speed-and-tweet#1]
By running the app with "node index.js" I get the following error:

```
running test
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'settings' of null
    at gotServers (C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\speedtest-net\index.js:423:18)
    at C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\speedtest-net\index.js:40:16
    at C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\speedtest-net\index.js:215:7
    at Parser.<anonymous> (C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\xml2js\lib\xml2js.js:489:18)
    at emitOne (events.js:77:13)
    at Parser.emit (events.js:169:7)
    at Parser.exports.Parser.Parser.parseString (C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\xml2js\lib\xml2js.js:499:16)
    at Parser.parseString (C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\xml2js\lib\xml2js.js:7:59)
    at exports.parseString (C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\xml2js\lib\xml2js.js:540:19)
    at C:\Users\myself\AppData\Roaming\npm\node_modules\speed-and-tweet\node_modules\speedtest-net\index.js:213:5
```

**Comment by tribet84** [comment]
By using this list of servers works fine:

http://c.speedtest.net/speedtest-servers-static.php?load=1

**Comment by tribet84** [comment]
You have a PR :)

**Comment by wasbazi** [comment]
Fixed with #4
